bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -337,6 +337,18 @@ def RDF(self):
         del type(self).RDF
         return ns
 
+    # Overload RooFit namespace
+    @property
+    def RooFit(self):
+        ns = self._fallback_getattr('RooFit')
+        try:
+            from .pythonization._roofit import pythonize_roofit_namespace
```

Comment:
Can we do this import before the try (i.e. at the very beginning of this property function)? This way if there is an exception in the try block we know for sure it comes from the pythonizor function.